libbeat/processors/cache/cache.go

```diff
@@ -133,6 +126,8 @@ type CacheEntry struct {
 
 // Run enriches the given event with the host metadata.
 func (p *cache) Run(event *beat.Event) (*beat.Event, error) {
+	p.SetPaths(paths.Paths) // set default if paths is not initialized
```

Comment:
Do we *have* to do this?  This means if we get the order wrong we default to all the beats in the process using the same paths which is wrong.